src/response_formaters.py

```diff
@@ -45,17 +45,24 @@ async def get_tourist_sites(session: AsyncSession, site_type: AllSitesFilter, vi
             if site.get('id') in stamp_places_id:
                 continue
 
-        # FIXME image will fail if there are no images to the place
+        # Fetch image id
+        image_id = (
+            await Images.all_by_place(
+                session, site.get('id'),
+            )
+        )
+
+        # Get first element of array only after we know we have an image
+        if image_id is not None:
+            image_id = image_id[0]
```

Comment:
Not length checked before usage! The method will **always** return a list but it **can** be empty!

Reply:
A proper fix will be a request that gets the first element only

Reply:
I will include it in the TODO list